source/assets/stylesheets/styles/_footer.scss

```diff
@@ -0,0 +1,159 @@
+.footer {
+  background-color: $dark-gray;
+  min-height: 280px;
+}
+
+.footer-container {
+  @extend %container;
+  box-sizing: border-box;
+  color: $white;
+  font-size: 0.85em;
+  padding: 30px 16px 50px;
+
+  @include breakpoint($tablet-breakpoint) {
+    padding: 50px 0;
+  }
+}
+
+.footer-container__address-tile {
+  border-bottom: groove 1px;
+  box-sizing: border-box;
+  display: block;
+  font-weight: 100;
+  line-height: 1.5em;
+  padding-bottom: 12px;
+  padding-right: 16px;
+  width: 100%;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    border-bottom: none;
+    padding-bottom: 40px;
+    padding-left: 16px;
+    width: 33.33%;
+  }
+}
+
+.footer-container__address-city {
+  font-weight: 400;
+  padding: 15px 0 10px;
+
+  a {
+    color: inherit;
+    text-decoration: none;
+  }
+
+  &:after {
+    content: ">";
+    padding-left: 8px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    padding-bottom: 25px;
+    padding-top: 0;
+
+    span {
+      display: none;
+    }
+
+    &:after {
+      content: ""
+    }
+  }
+}
+
+.footer-container__address-phone {
+  font-weight: 400;
+  color: inherit;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    pointer-events: none;
+    cursor: text;
+  }
+}
+
+.footer-container__address-full {
+  display: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: block;
+  }
+}
+
+.footer-container__registration-address {
+  display: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    font-size: 0.8em;
+    line-height: 1.4em;
+  }
+}
+
+.footer-container__company-name {
+  font-weight: 400;
+}
+
+i.footer__map-icon {
+  padding-right: 10px;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: none;
+  }
+}
+
+i.footer__phone-icon {
+  padding-right: 8px;
+
+  @include breakpoint($tablet-breakpoint) {
+    display: none;
+  }
+}
+
+.footer-container__newsletter {
+  font-weight: 100;
+
+  @include breakpoint($tablet-breakpoint) {
+    padding-left: 16px;
```

Comment:
Two things here:

1. I think we have quite a mixture of padding dimensions, but we should try and be consistent with them - I've been trying to use 20px left and right as a minimum after realising this (I have also update the project story styles on the `homepage` branch). Currently, on those pages this footer fits nicely with the related content above (which is also 16px), but on other pages such as the blog, the padding is 20px.

2. Secondly is because we only have a left padding, declaring 50% for the newsletter is showing larger than 50% (maybe 51/52%) because there is no padding on the right. Because of that the newsletter looks ever-so-slightly off. Hopefully the screenshot will be able to explain.

<img width="843" alt="screen shot 2015-11-20 at 15 28 33" src="https://cloud.githubusercontent.com/assets/885223/11303685/063975c4-8f9c-11e5-8aec-fa1fba625bb2.png">